### ISSUE TITLE
Replace Butterknife with ViewBinding

### ISSUE THREAD

**Issue by LightDestory** [LightDestory/FastHub-RE#1]
As the title says, Butterknife is deprecated and should be replaced

**Comment by LightDestory** [comment]
@zeromake performed such amazing step!

Closing it thanks to 46246e003362198f575d4824d02e848c57b84e8f

**Comment by zeromake** [comment]
@LightDestory 
I was just Butterknife replace to `View.findViewById`, use ViewBinding some Fragment top and left nav is will become blank.

**Comment by LightDestory** [comment]
> @LightDestory I was just Butterknife replace to `View.findViewById`, use ViewBinding some Fragment top and left nav is will become blank.

Yes, Iread the commit changes, I will try to check this issue again.

Did you receive my email? I sent to you like 2 days ago.

**Comment by zeromake** [comment]
@LightDestory 
Because I'm in China, you know, because of network problems, emails have to be read manually
